packages/web-components/src/player-component/UI/ui.class.ts

```diff
@@ -201,6 +201,61 @@ export class OverflowMenu extends shaka.ui.OverflowMenu {
     }
 }
 
+export class MetaDataButton extends shaka.ui.OverflowMenu {
+    private svg: SVGSVGElement;
+    private path: SVGPathElement;
+    private isBoxOn: boolean;
+    private isAttributesOn: boolean;
+
+    public constructor(parent: any, 
+                       controls: any, 
+                       private showBoxCallBack: (isOn: boolean) => void, 
+                       private showAttributesCallBack: (isOn: boolean) => void) {
+        super(parent, controls);
+        this.init();
+    }
+
+    public createChildren_() {
+        const showBoundingBox = document.createElement('div');
+        showBoundingBox.innerText = ' Bounding Box ';
+        const BoxCheckbox = document.createElement('fast-checkbox');
+        this.eventManager.listen(BoxCheckbox, 'click', () => {
+            this.isBoxOn = !this.isBoxOn;
+            this.showBoxCallBack(this.isBoxOn);
+        });
+        showBoundingBox.appendChild(BoxCheckbox);
+        this.overflowMenu_.appendChild(showBoundingBox);
+
+        const showAtrribute = document.createElement('div');
+        showAtrribute.innerText = ' Attributes ';
```

Comment:
>Attributes [](http://example.com/codeflow?start=36&length=10)

same here.. from localizatoin.. add it to en.json and de.json and use it from localization values